app/command_utils_browsertest.cc

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "brave/app/command_utils.h"
+
+#include "base/test/scoped_feature_list.h"
+#include "brave/components/commands/common/features.h"
+#include "chrome/browser/ui/browser_commands.h"
+#include "chrome/test/base/in_process_browser_test.h"
+#include "chrome/test/base/ui_test_utils.h"
+#include "content/public/test/browser_test.h"
+
+class CommandUtilsBrowserTest : public InProcessBrowserTest {
+ public:
+  void SetUp() override {
+    features_.InitAndEnableFeature(commands::features::kBraveCommandsFeature);
+    InProcessBrowserTest::SetUp();
+  }
+
+ private:
+  base::test::ScopedFeatureList features_;
+};
+
+IN_PROC_BROWSER_TEST_F(CommandUtilsBrowserTest,
+                       AllCommandsShouldBeExecutableWithoutCrash) {
+  ASSERT_TRUE(ui_test_utils::NavigateToURL(browser(), GURL("chrome://newtab")));
+  auto commands = commands::GetCommands();
+  for (const auto& command : commands) {
+    LOG(ERROR) << "Executing command: " << commands::GetCommandName(command);
+    chrome::ExecuteCommand(browser(), command);
+    LOG(ERROR) << "Executed!";
```

Comment:
I actually added these after I got the test working - it's quite handy in the test because ERROR logs are only printed if the test fails and it let's you know exactly which command failed.

Reply:
> t's quite handy in the test because ERROR logs are only printed if the test fails

Oh, that's cool! didn't know that. Then how about lowering the verbose level to INFO? I seems to work with that level too.

Reply:
> I actually added these after I got the test working - it's quite handy in the test because ERROR logs are only printed if the test fails and it let's you know exactly which command crashed.

that's not exactly correct. These logs are always printed to the stdout, but because we run tests in `--test-launcher-jobs=2` mode, you don't see it until the failure. If you run these tests with `--test-launcher-jobs=1`, then logs will appear.

The correct approach here is to write:
```
SCOPED_TRACE(testing::Message() << commands::GetCommandName(command));
```
This will add additional information during the test failure.

Reply:
Oh cool, I didn't know about that @goodov! Thanks